### ISSUE TITLE
Skip-CertificateCheck not working on v1.0.11.13236 Set-IntersightConfiguration

### ISSUE THREAD

**Issue by Jikkelsen** [CiscoDevNet/intersight-powershell#109]
**Describe the bug**
Skip-CertificateCheck not working on v1.0.11.13236 Set-IntersightConfiguration

**To Reproduce**
Steps to reproduce the behavior
- PowerShell code snippet (when possible)
```
 $onprem = @{
        BasePath             = "https://$Intersight"
        ApiKeyId             = $ApiKeyID
        ApiKeyFilePath       = $ApiKey
        HttpSigningHeader    = @('(request-target)', 'Host', 'Date', 'Digest')
        SkipCertificateCheck = $true
    }
Set-IntersightConfiguration @onprem
```

**Version used**
 - Intersight PowerShell Module version: v1.0.11.13236
 - Intersight Appliance Version: On-prem: 1.0.9-589

**Expected behavior**
I expect the -SkipCertificateCheck paramater to work. It does not seem to be working

**Comment by Ghufz** [comment]
Can you provide the error message , it would help us to point the issue. and also please provide the error message once you run with Set-IntersightConfiguration.
```powershell
$Error[0].Exception
```

**Comment by briamorr** [comment]
Seeing same issue as well against an appliance with the default self-signed certs:

```
PS C:\>  $onprem = @{
>>         BasePath             = "https://appliancefqdn"
>>         ApiKeyId             = "key/id/here"
>>         ApiKeyFilePath       = "C:/temp/SecretKey.txt"
>>         HttpSigningHeader    = @('(request-target)', 'Host', 'Date', 'Digest')
>>         SkipCertificateCheck = $true
>>     }
PS C:\> Set-IntersightConfiguration @onprem
Set-IntersightConfiguration: Error performing this operation. Invalid SSL Certificates. Use 'SkipCertificateCheck = $true' in configuration for invalid SSL certificates and retry the operation.

PS C:\> $Error[0].Exception

TargetSite     : Void ProcessRecord()
Message        : Error performing this operation. Invalid SSL Certificates. Use 'SkipCertificateCheck = $true' in configuration for invalid SSL certificates and
                 retry the operation.
Data           : {}
InnerException : Intersight.Client.ApiException: Error calling GetOrganizationOrganizationList: The SSL connection could not be established, see inner exception.
                    at Intersight.Api.OrganizationApi.GetOrganizationOrganizationListWithHttpInfo(String filter, String orderby, Nullable`1 top, Nullable`1 skip,
                 String select, String expand, String apply, Nullable`1 count, String inlinecount, String at, String tags, Int32 operationIndex)
                    at Intersight.PowerShell.SetConfigurationCmdlet.ProcessRecord() in /home/jenkins/workspace/Release-IS-Powershell-Module/src/github.com/cisco-inte
                 rsight/powershell-generator-intersight/generated/Intersight.PowerShell_1.0.11.13236/src/CmdletBase/ConfigurationCmdlet.cs:line 137
HelpLink       :
Source         : Intersight.PowerShell
HResult        : -2146233088
StackTrace     :    at Intersight.PowerShell.SetConfigurationCmdlet.ProcessRecord() in /home/jenkins/workspace/Release-IS-Powershell-Module/src/github.com/cisco-inte
                 rsight/powershell-generator-intersight/generated/Intersight.PowerShell_1.0.11.13236/src/CmdletBase/ConfigurationCmdlet.cs:line 153
                    at System.Management.Automation.CommandProcessor.ProcessRecord()


PS C:\> Get-IntersightOrganizationOrganization
Get-IntersightOrganizationOrganization: Error calling GetOrganizationOrganizationList: The SSL connection could not be established, see inner exception.
PS C:\> $Error[0].Exception

ErrorCode      : 0
ErrorContent   : The SSL connection could not be established, see inner exception.
Headers        :
TargetSite     : Void ProcessRecord()
Message        : Error calling GetOrganizationOrganizationList: The SSL connection could not be established, see inner exception.
Data           : {}
InnerException :
HelpLink       :
Source         : Intersight.PowerShell
HResult        : -2146233088
StackTrace     :    at Intersight.PowerShell.GetCmdletBase.ProcessRecord() in /home/jenkins/workspace/Release-IS-Powershell-Module/src/github.com/cisco-intersight/po
                 wershell-generator-intersight/generated/Intersight.PowerShell_1.0.11.13236/src/CmdletBase/GetCmdletBase.cs:line 115
                    at System.Management.Automation.CommandProcessor.ProcessRecord()
`

**Comment by Ghufz** [comment]
This issue has been addressed in the released version 1.0.11.13376
https://www.powershellgallery.com/packages/Intersight.PowerShell/1.0.11.13376

**Comment by Jikkelsen** [comment]
That was quick - thank you very much. Kind regards
